util/utils.py

```diff
@@ -36,12 +33,10 @@ def product_details(divs, list_of_product):
         
         product = {
             'name' : product_name.text,
-            'rating' : product_rating.text,
+            # 'rating' : product_rating.text,
```

Comment:
Why rating field is commented?
We need to store rating as well.

Reply:
fixed